content/legal-notices/evaluation-license/v2.md

```diff
@@ -0,0 +1,73 @@
+
+5.2	Ownership of all complete or partial copies of the Service, Supplementary Material and Information and related documentation shall at all times remain with Roadie or its licensors.
+
+5.3	If a third party notifies Customer of any claim that the use of the Service, Supplementary Material or Information infringes any right of a third party, Customer agrees to immediately notify Roadie. If any such claim is made to Customer or Roadie, Customer shall, at Roadie's request, immediately cease use of the Service, Supplementary Material and Information. If Roadie is unable to allow Customer to continue evaluation of the Service, the provisions of clause 3.2 shall apply.
+
+5.4	Customer created plug-ins or other content introduced to the Service during the Trial Period remains the property of Customer and, where Customer has not opted to purchase a full licence to the Service, will be deleted at the end of the Retention Period in accordance with the terms of clause 6.3. Customer is responsible for maintaining its own back-ups or copies of any data or content introduced to the Service during the Trial Period. Customer must own or hold valid licences for all content introduced to the Service by Customer.
+
+5.5	All rights not explicitly granted under this licence are reserved.
+
+### 6. Data Protection
+
+6.1	Roadie will act as a data controller for all personal data that is processed on the Service for the duration of the Evaluation Licence as it is expected that there will be limited personal data on the Service during the Trial Period. This data will be processed in accordance with our Privacy Policy and Cookies Policy available on our website.
+
+6.2	If Customer wishes to act as data controller for the Trial Period, with Roadie as data processor, please contact Roadie to request Roadie’s Data Processing Agreement.
+
+6.3 Roadie will retain Customer’s data, including personal data, for the Retention Period. Usage data collected during the trial may also be retained after the trial for the Retention Period. Customer data will be retained for the purpose of enabling the Customer to re-start the evaluation or convert to a full license without re-entering the data from the evaluation. Usage data will be retained for the purposes of improving and optimizing the Service and Website. At the end of this Retention Period, where the Customer has not opted to purchase a full licence, Roadie will delete all Customer data on the Service. After the Retention Period Usage data will be retained in pseudo anonymised form for this purpose. This clause overrides the terms of the Data Processing Agreement, if applicable.
```

Comment:
gap missing between 6.3 and start of paragraph

Reply:
spacing is missing too. it is supposed to be three sections so its easier to read. This is pretty dense and hard to read.

Reply:
I chose not to add a gap because no other term has a gap in it. Is the gap important?

Reply:
I'm concerned that it looks like a new clause with no number if we leave gaps.

Reply:
we don't have to add spacing in the paragraph, it just makes it easier to read if we do, but its not essential. 

But the other gap was this:
![Screenshot 2021-12-09 at 11 11 11](https://user-images.githubusercontent.com/42720161/145386031-349b09eb-afc4-4e45-a7d3-f74a218926ed.png)

Reply:
That gap is gone and/or doesn't render in the finished product. [Check the preview to see.](https://deploy-preview-511--roadie.netlify.app/legal-notices/evaluation-license/v2/)